### ISSUE TITLE
.deb generated with `--snapshot` is broken when the git tag starts with a `v`

### ISSUE THREAD

**Issue by yml** [goreleaser/goreleaser#584]
### Prerequisites

* [x] Are you running the latest version?
* [x] Did you perform a search to see if this problem was already reported?

For more information, see the `CONTRIBUTING` guide.

### Description

When you create a git tag starting with a `v` the generated .deb is invalid 

### Environment

Please paste the output of:

* `go version`
```
go version go1.10 linux/amd64
```
* `uname -a`
```
Linux kodi 4.10.0-42-generic #46-Ubuntu SMP Mon Dec 4 14:38:01 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

* `goreleaser --debug`
```
yml@kodi$ goreleaser --debug

   • releasing...
   • loading config file       file=.goreleaser.yml
   • loaded config file        config={gobby { %!s(bool=false) %!s(bool=false) } { { }     []  []   %!s(bool=false) } { { }   } [{[darwin linux windows] [] [] [] [] ./main.go   gobby {go generate } [CGO_ENABLED=0] }] { map[darwin:Darwin linux:Linux windows:Windows 386:i386 amd64:x86_64]  [{windows zip}] %!s(bool=true) []} { map[] [] [] [] [] []       map[] map[]} {{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }} map[darwin:macOS linux:linux amd64:64-bit 386:32-bit] [deb] [] [] [] [] Gwadeloop. https://github.com/lincolnloop/Gobby Yann Malet <yann.malet@gmail.com> Web server for Jobby-wrap in GO TBD /usr/local/bin map[] map[]} { map[]      map[]} {{{ .Tag }}_{{ .Commit}}} {{{ .ProjectName }}_{{ .Version }}_checksums.txt} [] [] {{[^docs: ^test:]} asc} ./dist { []  } {} {[] [] [] [] []     { } [] } {  }}
   • parallelism: 4
   • SETTING DEFAULTS FOR:
      • loading environment variables
      • snapshoting
      • releasing to GitHub
      • running git               args=[rev-parse --is-inside-work-tree]
      • git result                output=true

      • running git               args=[config --get remote.origin.url]
   ⨯ release failed after 0.01s error=repository doesn't have an `origin` remote
```

### Steps to Reproduce

1. Create a git tag starting with a `v`
```shell
$ git tag -a v0.0.3 -m "0.0.3: bug in goreleaser snapshot if the tag starts with a v"
$ git tag
0.1.0
v0.0.3
```

2. Generate the .deb using `goreleaser --snapshot --rm-dist`
3. Install this .deb

```shell
root@test-gobby:~# dpkg -i /tmp/gobby_v0.0.3_b10d29d694fbf3ec3452df21e819ea91f6d90b6a_linux_64-bit.deb 
dpkg: error processing archive /tmp/gobby_v0.0.3_b10d29d694fbf3ec3452df21e819ea91f6d90b6a_linux_64-bit.deb (--install):
 parsing file '/var/lib/dpkg/tmp.ci/control' near line 2 package 'gobby':
 error in 'Version' field string 'v0.0.3': version number does not start with digit
Errors were encountered while processing:
 /tmp/gobby_v0.0.3_b10d29d694fbf3ec3452df21e819ea91f6d90b6a_linux_64-bit.deb
```
**Expected behavior:** 

I expected to be able to install the .deb without error.

**Actual behavior:**

I received an error related to the version not starting with a digit.
```
error in 'Version' field string 'v0.0.3': version number does not start with digit
```

**Comment by ncw** [comment]
Possibly related to this https://github.com/goreleaser/nfpm/issues/9 ?

**Comment by ncw** [comment]
Also the go releaser packages have this problem on my ubuntu artful laptop

```
$ sudo dpkg -i /tmp/goreleaser_amd64.deb
dpkg: error processing archive /tmp/goreleaser_amd64.deb (--install):
 parsing file '/var/lib/dpkg/tmp.ci/control' near line 2 package 'goreleaser':
 error in 'Version' field string 'v0.57.0': version number does not start with digit
Errors were encountered while processing:
 /tmp/goreleaser_amd64.deb
```

**Comment by caarlos0** [comment]
fixed in nfpm, testing on goreleaser...

**Comment by caarlos0** [comment]
should be fixed in v0.57.1
thanks for reporting, was totally my bad!
